functional-tests/envs/testenv.py

```diff
@@ -226,22 +245,16 @@ def init(self, ctx: flexitest.EnvContext) -> flexitest.LiveEnv:
         if self.auto_generate_blocks:
             generate_blocks(brpc, BLOCK_GENERATION_INTERVAL_SECS, seqaddr)
 
-        rpc_port = bitcoind.get_prop("rpc_port")
-        rpc_sock = f"localhost:{rpc_port}/wallet/{walletname}"
-        bitcoind_config = BitcoindConfig(
-            rpc_url=rpc_sock,
-            rpc_user=bitcoind.get_prop("rpc_user"),
-            rpc_password=bitcoind.get_prop("rpc_password"),
-        )
-
         reth_config = RethELConfig(
             rpc_url=f"localhost:{reth_port}",
             secret=reth_secret_path,
         )
-        reth_rpc_http_port = reth.get_prop("eth_rpc_http_port")
 
+        # FIXME: sequencer doesn't require http:// but datatool does.
+        bitcoind_config.rpc_url = bitcoind_config.rpc_url.removeprefix("http://")
```

Comment:
Well you could change the `datatool` API to be inline with the sequencer in this PR. It is much much much better than python string manipulation glue code with a `FIXME` comment.
Also if you do that, don't forget to update the `datatool` testing/docs/README etc.